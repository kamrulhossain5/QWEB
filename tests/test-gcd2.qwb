function int gcd(int a, int b){
  REPEAT until (a != b){
    if (a > b){
      a = a - b;
    }
    otherwise{
      b = b - a;
    }
  }
  output a;
}

function int main(){
  print(gcd(14,21));
  print(gcd(8,36));
  print(gcd(99,121));
}
