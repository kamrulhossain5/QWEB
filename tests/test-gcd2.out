7<br>
4<br>
11<br>
